### PR TITLE
Thr 28 docker static h5p files

### PR DESCRIPTION
**Pull request by MajedAlaitwniCap**
## Description 
- Repo with Docker container for hosting h5p Static Files
- add GitHub workflow to apply run the Jobs in GitHub Action
- add ansible for h5p-staticfiles-server to be deployed on the Cluster

## Links to Tickets or other pull requests:
- [THR-28](https://ticketsystem.dbildungscloud.de/browse/THR-28)
- https://github.com/hpi-schul-cloud/dof_app_deploy/pull/573

## Changes
the PR necessary to add the container hosting h5p static files on the Cluster

## New Repo:
[h5p-staticfiles-server](https://github.com/hpi-schul-cloud/h5p-staticfiles-server)

## New Repo:
this is a new Repo

### CODE REVIEW

File: ansible/group_vars/all/h5p-statics.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+H5P_STATICFILES_SERVER_IMAGE: quay.io/schulcloudverbund/h5p-staticfiles-server
```
Comment (mamutmk5) [inline ansible/group_vars/all/h5p-statics.yml]: This line add the same variable to ansible as in ansible/group_vars/all/h5p-staticfiles-server.yml

File: ansible/group_vars/all/h5p-statics.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+H5P_STATICFILES_SERVER_IMAGE_TAG: 0.1.0
```
Comment (mamutmk5) [inline ansible/group_vars/all/h5p-statics.yml]: this must be to dof_app_deploy and then to the ansible host_vars for each instance at the version.yml

File: ansible/roles/h5p-staticfiles-server-core/templates/configmap.yml.j2 [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  NODE_ENV: "production"
+  CORS_ORIGIN: "http://h5p-staticfiles-server-svc:8080"
+  # Paths
+  SCHULCLOUD_BASE_PATH: "http://api-svc:3030/api/v1"
+  SCHULCLOUD_ALL_USERS_FOR_UUID: "http://api-svc:3030/api/v1/resolve/users/"
+  SCHULCLOUD_ALL_SCOPES_FOR_TOKEN: "http://api-svc:3030/api/v1/resolve/scopes/"
+  TZ: "Europe/Berlin"
+{% if CAL_IS_MIGRATION is defined %}
+  IS_MIGRATION: "{{ CAL_IS_MIGRATION }}"
+{% endif %}
```
Comment (mamutmk5) [inline ansible/roles/h5p-staticfiles-server-core/templates/configmap.yml.j2]: Where are this Enviorment variable's used at the running container?

File: nginx.conf [hunk 4]
```diff
@@ -0,0 +1,22 @@
+events { 
+    worker_connections 1024;
+}
```
Comment (mamutmk5) [inline nginx.conf]: Not necessary to set it

**Review by mamutmk5 — CHANGES_REQUESTED** [review]

File: ansible/group_vars/all/h5p-statics.yml (hunk 2 above)
Comment (mamutmk5) [reply ansible/group_vars/all/h5p-statics.yml]: The `H5P_STATICFILES_SERVER_IMAGE_TAG` is the topic from my comment

File: ansible/roles/h5p-staticfiles-server-core/templates/configmap.yml.j2 (hunk 3 above)
Comment (MajedAlaitwniCap) [reply ansible/roles/h5p-staticfiles-server-core/templates/configmap.yml.j2]: not at all on this moment deleted

**Review by mamutmk5 — APPROVED** [review]

**Pull request merged**